Makefile

```diff
@@ -400,31 +402,18 @@ cluster-clean: build-manifest ## Undeploy Kepler in the cluster.
 .PHONY: cluster-clean
 
 cluster-deploy: ## Deploy Kepler in the cluster.
-	BUILD_CONTAINERIZED=$(BUILD_CONTAINERIZED) \
 	./hack/cluster-deploy.sh
 .PHONY: cluster-deploy
 
 cluster-up:  ## Create the Kind cluster, with Prometheus, Grafana and Kepler
-	CLUSTER_PROVIDER=$(CLUSTER_PROVIDER) \
```

Comment:
Are we happy that there are reasonable defaults configured for all these params in `./hack/cluster.sh`
in my workflow I run a lot of these make targets directly, is it expected that I now set these variables when I run the make target?

Reply:
nvm I see the export above. LGTM